specs/duvet-specification.md

```diff
@@ -0,0 +1,475 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Duvet specification
+
+## Version
+
+0.2.0
+
+### Changelog
+
+- 0.2.0
+
+  - Initial record
+
+- 0.1.0
+
+  - "Specless" Rust Implementation
+
+## Overview
+
+This document introduces and describes Duvet.
+
+Any implementation of Duvet MUST follow this specification.
+
+## Introduction
+
+Duvet is an application to build confidence that your software is correct.
+The first step in correct software is to document what correct behavior is.
+This document is called a specification.
+A specification can be a design document or an RFC.
+This specification document describes an application’s behaviors.
+What steps are important, in what order and why.
```

Comment:
It's definitely an incomplete sentence. We're missing a subject and verb. It can be fixed with something like

```suggestion
It includes what steps are important, in what order, and why.
```

Reply:
```suggestion
It includes which steps are important, in what order, and why.
```